runConfigurations/Cromwell_server.run.xml

```diff
@@ -18,4 +19,4 @@
       <option name="RunConfigurationTask" enabled="true" run_configuration_name="renderCiResources" run_configuration_type="SbtRunConfiguration" />
     </method>
   </configuration>
-</component>
```

Comment:
Newline needed

Reply:
actually it's complaining that I added one because I foolishly edited the file manually 🙄